src/platform/Ameba/ConfigurationManagerImpl.cpp

```diff
@@ -36,9 +36,25 @@ namespace DeviceLayer {
 
 using namespace ::chip::DeviceLayer::Internal;
 
-/** Singleton instance of the ConfigurationManager implementation object.
+/** Singleton pointer to the ConfigurationManager implementation.
  */
-ConfigurationManagerImpl ConfigurationManagerImpl::sInstance;
+ConfigurationManager * gInstance;
+
+ConfigurationManager & ConfigurationMgr()
+{
+    return *gInstance;
```

Comment:
ChipVerifyOrDie(gInstance != nullptr) may be useful here just in case.

Reply:
I've added that check.